src/renderer/system/utils.js

```diff
@@ -1,10 +1,23 @@
 import path from 'path';
 import { clipboard, remote, shell } from 'electron';
+import ms from 'ms';
+
+const __cache = {
+  timer: null
+};
 
 const currentWindow = remote.getCurrentWindow();
 
 export function copyToClipboard(text) {
   clipboard.writeText(text);
```

Comment:
You need to clear the cache here. Similar to what happens in the other file?
https://github.com/buttercup/buttercup-desktop/blob/9f7fd929ae5c14986a12775f646d8a2135081797/src/renderer/system/shortcuts.js#L16

Reply:
Hey, thank you for the reply!  
I copied the function from shortcut.js to utils.js exactly, so I am not quite sure as to what you are referring to in regards to clearing the cache similar to the other file?  Thanks !

Reply:
Hi! If you check `shortcut.js` there is a few more lines in the beginning of the function which you need to copy as well

Reply:
Thanks, I will take a look.

Reply:
So I added in a bit more in addition to clearing the cache.  Basically what I overlooked before was that the method is being called by every "copy-button" not just the password and this caused the clipboard to be cleared after 15 seconds regardless of whether it was a password or not.

This new change basically ensures that only the password will be cleared from the clipboard after 15 seconds.

I hope this helps 

Let me know your thoughts on the matter , thanks !